### ISSUE TITLE
Bad log message on peer move

### ISSUE THREAD

**Issue by bboreham** [weaveworks/weave#1106]
````
weave 2015/07/09 08:07:22.804784 Peer da:08:ed:d4:49:31(boot2docker) moved from true to 80.152.241.31:58782
````

The `true` should be an IP endpoint.

**Comment by bboreham** [comment]
Fixed via cbcd5da

**Comment by rade** [comment]
When was this introduced? Did it ever appear in a release?

**Comment by errordeveloper** [comment]
This is from user logs, which are from a 1.0.1.

**Comment by rade** [comment]
ah. really should have fixed it on 1.0 then, especially since the fix is trivial.

**Comment by bboreham** [comment]
Broken in https://github.com/weaveworks/weave/commit/27c9e70ea9cb900339282cad6527b1e6f29dcb35

**Comment by bboreham** [comment]
Fixed for 1.0 in b93070f91ad1222404a330922fe0761ee05ae0b5
